sklearn/metrics/pairwise.py

```diff
@@ -1566,6 +1566,13 @@ def additive_chi2_kernel(X, Y=None):
     return result
 
 
+@validate_params(
+    {
+        "X": ["array-like", "sparse matrix"],
```

Comment:
X and Y are not needed as they are validated [here](https://github.com/scikit-learn/scikit-learn/blob/e0ffa0ef9b38f997b7cf2a8e1f0e675d0b4a4d9e/sklearn/metrics/pairwise.py#L1509).

I got some of this feedback last week.

Reply:
Hi,

Thanks for the review, made changes to remove X and Y validations.

Reply:
Hi,

Pytest wont pass for validations, also X and Y are not validated for sparse matrices, so this validation even in repeat is needed.

Reply:
We need to validate `X` and `Y` otherwise the name of the error will not be OK. We decided to use another way to avoid nested checks.